doc/getting-started.rst

```diff
@@ -0,0 +1,236 @@
+Getting Started
+===============
+
+This guide will walk you through setting up your development environment, building and flashing your first firmware.
+
+Prerequisites
+-------------
+
+If you're new to Zephyr, we recommend starting with the official
+`Zephyr Getting Started Guide <https://docs.zephyrproject.org/latest/develop/getting_started/index.html>`_
+to familiarize yourself with the framework.
+
+Before continuing, ensure you have installed the
+`Zephyr system requirements <https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-dependencies>`_
+for your operating system.
```

Comment:
These instructions were in the README and I moved them over, but would it be clearer if we remove these links and just have all the setup in this doc?